src/main/java/ca/on/oicr/pineryreports/reports/impl/GazpachoProjectStatusReport.java

```diff
@@ -0,0 +1,617 @@
+        if (mostRecent == null) return Long.toString(daysSinceReceived);
+        try {
+          runCompleted = getDateTimeFormat().parse(mostRecent.getCompletionDate());
+        } catch (java.text.ParseException e) {
+          System.out.println("Error parsing completion date for run " + mostRecent.getId());
+          return "";
+        }
+        long daysReceivedToCompleted = Duration.between(receivedByLab.toInstant(), runCompleted.toInstant()).toDays();
+        return Long.toString(daysReceivedToCompleted);
+      }
+    }
+
+    @Override
+    public int compareTo(ReportItem other) {
+      return this.getStock().getName().compareTo(other.getStock().getName());
+    }
+  }
+
+  private static final Instant TODAY = ZonedDateTime.now().toInstant();
+  private static final Instant TWO_DAYS_AGO = ZonedDateTime.now().minusDays(2).toInstant();
+
+  private String project;
+  private String start;
+  private String end;
+  private String analyte;
+  private final String todate = removeTime(TODAY.toString());
+
+  private static final Option OPT_PROJECT = CommonOptions.project(true);
+  private static final Option OPT_AFTER = CommonOptions.after(true);
+  private static final Option OPT_BEFORE = CommonOptions.before(true);
+  private static final Option OPT_ANALYTE = CommonOptions.analyte(false);
+  public static final String REPORT_NAME = "project-status";
+
+  // keep track of the base stock name for all samples/libraries/runs active this month
+  Map<String, ReportItem> dnaThisMonth = Maps.newTreeMap();
+  Map<String, ReportItem> rnaThisMonth = Maps.newTreeMap();
+  Map<String, ReportItem> allDna = Maps.newTreeMap();
+  Map<String, ReportItem> allRna = Maps.newTreeMap();
+  List<Map.Entry<String, ReportItem>> dnaRecentList;
+  List<Map.Entry<String, ReportItem>> rnaRecentList;
+  List<Map.Entry<String, ReportItem>> dnaAllList;
+  List<Map.Entry<String, ReportItem>> rnaAllList;
+
+  @Override
+  public String getReportName() {
+    return REPORT_NAME;
+  }
+
+  @Override
+  public Collection<Option> getOptions() {
+    return Sets.newHashSet(OPT_PROJECT, OPT_AFTER, OPT_BEFORE, OPT_ANALYTE);
+  }
+
+  @Override
+  public void processOptions(CommandLine cmd) throws ParseException {
+    if (cmd.hasOption(OPT_AFTER.getLongOpt())) {
+      String after = cmd.getOptionValue(OPT_AFTER.getLongOpt());
+      if (!after.matches(DATE_REGEX)) {
+        throw new ParseException("After date must be in format yyyy-mm-dd");
+      }
+      this.start = after;
+    }
+
+    if (cmd.hasOption(OPT_BEFORE.getLongOpt())) {
+      String before = cmd.getOptionValue(OPT_BEFORE.getLongOpt());
+      if (!before.matches(DATE_REGEX)) {
+        throw new ParseException("Before date must be in format yyyy-mm-dd");
+      }
+      this.end = before;
+    }
+
+    if (cmd.hasOption(OPT_ANALYTE.getLongOpt())) {
+      String analyte = cmd.getOptionValue(OPT_ANALYTE.getLongOpt());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename "analyte" which hides the field declared at line 222. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AHiddenFieldCheck)